src/pages/index.tsx

```diff
@@ -1,17 +1,45 @@
-import React from 'react';
-import Link from 'next/link';
-import { Button, Flex, Input } from '@blockstack/ui';
-import { BehaviorSubject } from 'rxjs';
+import React, { useState } from 'react';
+import Router from 'next/router';
+import { Flex, Text } from '@blockstack/ui';
 
-const searchTerm$ = new BehaviorSubject('');
+import { Page } from '@components/page';
+import { SearchBar } from '@components/search-bar';
+import { HomeNav } from '@components/home-nav';
+import { DevLinks } from '@components/dev-links';
+import { useRecentlyViewed } from '../common/hooks/use-recently-viewed';
 
-export const Home = () => (
-  <Flex flexDirection="column" height="100vh" width="100%" align="center" justify="center">
-    <Link href="/components">
-      <Button>Components</Button>
-    </Link>
-    <Input type="text" onChange={(e: any) => searchTerm$.next(e.target.value)} />
-  </Flex>
-);
+export const Home = () => {
+  const [query, setQuery] = useState('');
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    if (!query.trim().length) return;
+    await Router.push('/txid/' + query);
+  };
+
+  const recentlyViewedTransactions = useRecentlyViewed();
+
+  return (
+    <Page>
+      {JSON.stringify(recentlyViewedTransactions, null, 2)}
+      <HomeNav />
+      <Flex
+        as="form"
+        flexDirection="column"
+        align="center"
+        maxWidth="544px"
+        mt="20vh"
+        justify="center"
+        onSubmit={handleSubmit}
+      >
+        <Text as="h1" fontSize="36px" display="block" width="100%" textAlign={['center', 'left']} mb={6}>
```

Comment:
It would be good to start using the named spacing units we've added, in place of numbers for margin/padding: https://github.com/blockstack/ux/blob/master/packages/ui/src/theme/sizes.tsx#L30